src/osd/osd_types.h

```diff
@@ -3986,7 +3986,7 @@ struct object_info_t {
 
   flag_t flags;
 
-  static string get_flag_string(flag_t flags) {
+  string get_flag_string(flag_t flags) const {
```

Comment:
@tchaikov If this way, shall we change the function  "string get_flag_string() const;" to "static string get_flag_string() ;"?  This function will call  get_flag_string(flag)